keras/callbacks.py

```diff
@@ -547,6 +547,10 @@ def on_epoch_end(self, epoch, logs=None):
         send['epoch'] = epoch
         for k, v in logs.items():
             send[k] = v
+        if isinstance(v, (np.float16, np.float32)):
+            send[k] = np.float64(v)
+        elif isinstance(v, (np.int16, np.int32)):
+            send[k] = np.int64(v)
```

Comment:
```python
if isinstance(v, np.ndarray):
    send[k] = v.item()
else:
    send[k] = v
````

Reply:
changed as per suggestion,Travis build failed due to some tensorflow installation error,I guess re-running the test will pass the PR